lib/shared/analytics.js

```diff
@@ -90,10 +66,30 @@ analytics.service('AnalyticsService', function() {
    * @param {Error} exception - exception
    *
    * @example
-   * AnalyticsService.logException(new Error('Something happened'));
+   * analytics.logException(new Error('Something happened'));
    */
-  this.logException = resinCorvus.logException;
+  logException: resinCorvus.logException
+
+};
+
+/**
+ * @returns {Object} analytics module
+ */
+const runOnce = () => {
+  resinCorvus.install({
```

Comment:
Let's move this to the top of the file. By the way CommonJS works, this file should only be actually interpreted once. The interpreted result is then cached and returned every time the file is required.